src/Bicep.Core/Semantics/BuiltInNamespaceSymbol.cs

```diff
@@ -1,18 +1,19 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
+using Bicep.Core.TypeSystem;
 using Bicep.Core.TypeSystem.Types;
 
 namespace Bicep.Core.Semantics
 {
     public class BuiltInNamespaceSymbol : Symbol, INamespaceSymbol
     {
-        public BuiltInNamespaceSymbol(string name, NamespaceType type)
+        public BuiltInNamespaceSymbol(string name, TypeSymbol type)
```

Comment:
Built in namespaces can now have `ErrorType` as a type, to communicate diagnostics (e.g. restoration failures for an implicit provider).